aws_lambda.py

```diff
@@ -1,53 +1,23 @@
 #!/usr/bin/env python
-import base64
-import concurrent.futures
-import copy
-import hashlib
-import inspect
+import glob
+import importlib
 import os
-import requests
-import shutil
 import sys
-import time
-from datetime import datetime
-from tempfile import mkdtemp
 
-import boto3
-import boto3.session
-import cryptography
-import cryptography.x509
-import ecdsa
-from amo2kinto.generator import main as generator_main
-from botocore.exceptions import ClientError
-from cryptography.hazmat.backends import default_backend as crypto_default_backend
-from cryptography.hazmat.primitives import serialization as crypto_serialization
-from cryptography.x509.oid import NameOID
-from kinto_http import Client, KintoException
-from kinto_signer.serializer import canonical_json
 
-
-PARALLEL_REQUESTS = 4
-
-
-def command(func):
-    """Decorator to mark functions of this module as a CLI *command* for the help output.
-    """
-    func.__is_command = True
-    return func
-
-
-@command
 def help(**kwargs):
     """Show this help.
     """
 
     def white_bold(s):
         return f"\033[1m\x1B[37m{s}\033[0;0m"
 
+    entrypoints = [
+        os.path.splitext(os.path.basename(f))[0] for f in glob.glob("./commands/[a-z]*.py")
```

Comment:
```suggestion
        os.path.splitext(os.path.basename(f))[0]
        for f in glob.glob("./commands/[a-z]*.py")
```

for `black` to be happy.